### PR TITLE
Add Schema.org JSON-LD Microdata for SEO (FAQ page)

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
Following #290 

As specified on [https://developers.google.com/search/docs/data-types/faqpage](https://developers.google.com/search/docs/data-types/faqpage)

Final result:
image

### CODE REVIEW

**Comment by RiccardoMasutti** [comment]
You can check the code here: [https://search.google.com/test/rich-results?utm_campaign=devsite&utm_medium=microdata&utm_source=faq-page&id=Za9deuDEffxEdjHebtcHQw](https://search.google.com/test/rich-results?utm_campaign=devsite&utm_medium=microdata&utm_source=faq-page&id=Za9deuDEffxEdjHebtcHQw)

**Comment by m52go** [comment]
@RiccardoMasutti thanks! Two things.

1. Could you please add these improvements to the non-English `faq.html` files?

2. Could you indent the new `div` blocks to be consistent with the rest of the document?

Example:
```
  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="why-security-deposit" itemprop="name">{{ item.hSecurityDeposit }}</h6>
  <div class="accordion-content pb-5 col-sm-12 col-md-8 ml-0 pl-sm-0 pl-md-0 px-0" itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
    <div itemprop="text">
    {{ item.pSecurityDeposit }}
    </div>
  </div>
  </div>
```

Suggested:
```

  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="why-security-deposit" itemprop="name">{{ item.hSecurityDeposit }}</h6>
    <div class="accordion-content pb-5 col-sm-12 col-md-8 ml-0 pl-sm-0 pl-md-0 px-0" itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
      <div itemprop="text">
      {{ item.pSecurityDeposit }}
      </div>
    </div>
  </div>
```

**Comment by RiccardoMasutti** [comment]
@m52go Done

File: es/faq.html [hunk 1]
```diff
@@ -130,52 +130,269 @@ <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="dao-for-traders">{{
 
   <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="without-bsq">{{ item.hUseBSQ }}</h6>
   <div class="accordion-content pb-5 col-sm-12 col-md-8 ml-0 pl-sm-0 pl-md-0 px-0">
-      {{ item.pUseBSQ }}
+<h1>{{ item.hFAQ }}</h1>
```
Comment (m52go) [inline es/faq.html]: Looks like lines 11 through 132 are included by mistake?

**Review by m52go — CHANGES_REQUESTED** [review]
Thanks for the updates. One item remains...please see my comment.

File: es/faq.html (hunk 1 above)
Comment (RiccardoMasutti) [reply es/faq.html]: Yes it was a mistake. Fixed.

**Comment by m52go** [comment]
ACK

**Pull request merged**